homeassistant/helpers/event.py

```diff
@@ -1185,7 +1185,7 @@ def _refresh(
         """
         updates: list[TrackTemplateResult] = []
         info_changed = False
-        now = event.time_fired if not replayed and event else dt_util.utcnow()
+        now = event.time_fired_timestamp if not replayed and event else time.time()
```

Comment:
Should this be using monotonic?

Reply:
Ideally it would but since we fire events with wall clock time we need to use time here